cg/meta/workflow/analysis.py

```diff
@@ -415,6 +424,17 @@ def link_fastq_files_for_sample(
             self.fastq_handler.concatenate(linked_reads_paths[read], concatenated_paths[read])
             self.fastq_handler.remove_files(value)
 
+    def get_target_bed(self, case_id: str, analysis_type: str) -> str:
+        """
+        Call the extraction of the target bed file from lims and use default capture kit for WGS
+        """
+        target_bed: str = self.get_target_bed_from_lims(case_id=case_id)
+        if not target_bed:
+            if analysis_type == AnalysisType.WHOLE_GENOME_SEQUENCING:
+                return DEFAULT_CAPTURE_KIT
+            raise ValueError("No capture kit was found in LIMS")
+        return target_bed
```

Comment:
I would move this method to the RareDisease analysis API since it's specific to this workflow